src/components/Version.js

```diff
@@ -25,16 +25,14 @@ class Version extends React.Component {
    * If it's software wallet show modal warning
    */
   walletTypeClicked = () => {
-    if (LEDGER_ENABLED) {
-      if (!LOCAL_STORE.isHardwareWallet()) {
-        $('#softwareWalletWarningModal').modal('show');
-        this.context.showModal(MODAL_TYPES.ALERT, {
-          body: <SoftwareWalletWarningMessage />,
-          buttonName: 'Ok',
-          id: 'softwareWalletWarningModal',
-          title: 'Software wallet warning',
-        });
-      }
+    if (!LOCAL_STORE.isHardwareWallet()) {
+      $('#softwareWalletWarningModal').modal('show');
```

Comment:
We should refactor this modal to be a GlobalModal, to prevent the "greyed screen" bug

Should we do it in this PR?

Reply:
It already is, this line is doing nothing actually.
I removed it and the warning still works